### DISCUSSION TITLE
489

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#921] [Comments]
# 489

Spring Batch는 아직까지 확실한 표준 관리 도구가 없습니다. 각 팀/회사마다 상이한 방법들을 사용하는데요. 대표적인 방법들은 아래와 같습니다. Cron 클라우드 서버리스 이를테면 AWS의 Lambda 에 배치 코드를 등록하고, AWS CloudWatch에서 스케줄링 실행을 하도록 구성하는 방식입니다. API 기반의 직접 만든 관리자 페이지 Spring Batch Admin Deprecated 되었습니다. 더이상 개선하지 않겠다고 합니다. Spring Cloud Data Flow 로 전환하라고 합니다. 참고 Quartz를 이용한 관리자 페이지 Scheduler 역할로 Quartz를 사용하고 그에 대한 UI 대시보드를 직접 만드는 경우입니다. 참고 CI 서비스 (Jenkins / Teamcity 등..

https://jojoldu.tistory.com/489

**Comment by jhoonkim92** [comment]
안녕하세요 spring batch관련 좋은글들 잘 보고있습니다.
혹시 지금시점에서 jenkins대신 teamcity나 spring cloud로 관리도구를 사용하고 계신게 있으실까요?
해당글 작성으로부터 3년이 넘은 시간이라 궁금합니다!

**Comment by namhokim** [comment]
"Spring Cloud Data Flow"는 2025년 4월 21일 오픈 소스 프로젝트에서 종료가 되었다고 발표했군요.
이제 Tanzu Spring 고객에게만 상업적으로 제공하므로 무료로 사용하는 고객은 앞으로 사용이 제한되겠네요.
ref. https://spring.io/blog/2025/04/21/spring-cloud-data-flow-commercial
